jerry-core/ecma/operations/ecma-regexp-object.cpp

```diff
@@ -1003,12 +1003,13 @@ re_match_regexp (re_matcher_ctx_t *re_ctx_p, /**< RegExp matcher context */
           ecma_completion_value_t match_value = re_match_regexp (re_ctx_p, bc_p, str_p, &sub_str_p);
           if (!ecma_is_value_true (match_value))
```

Comment:
Not related to this PR:
The `ecma_is_value_true` call is not correct for the `match_value` as the function expects `ecma_value_t`, but the variable is an `ecma_completion_value_t`.